Menu.py

```diff
@@ -33,13 +35,17 @@ def exibicao_de_categorias():
 			elif escolha == 0:
 				break
 			else:
-				instalacao_de_pacotes(escolha, categorias, instalador, argumento_do_instalador)
+				if len(itens_escolha) > 1:
+					for instalacao in itens_escolha:
+						instalacao_de_pacotes(instalacao, categorias, instalador, argumento_do_instalador)
+				else:
+					instalacao_de_pacotes(escolha, categorias, instalador, argumento_do_instalador)
```

Comment:
Se o usuário passar 1 argumento, o split vai continuar trazendo uma lista.
Assim, vai quebrar a aplicação. (Eu removeria o for de dentro do IF/ELSE e um abraço para o gaiteiro)

Reply:
Alterei um pouco o código.
Depois vê se ficou melhor ou se prefere alterar algo mais.